server/src/main/java/org/opensearch/index/snapshots/blobstore/RemoteStoreShardShallowCopySnapshot.java

```diff
@@ -0,0 +1,324 @@
+/*
```

Comment:
nit: remove this file from this PR.

Reply:
Added a new method `asClone` to this existing file which is part of [Create snapshot API](https://github.com/opensearch-project/OpenSearch/pull/7118) PR.